examples/jsblocks/readme.md

```diff
@@ -0,0 +1,27 @@
+# jsblocks TodoMVC Example
+
+ > From simple user interfaces to complex single-page applications using faster, server-side rendered and easy to learn framework
+ >
+ > [jsblocks - jsblocks.com](http://jsblocks.com)
+
+
+## Resources
+
+- [Website](http://jsblocks.com)
+- [Documentation](http://jsblocks.com/learn)
+- [Download](http://jsblocks.com/download)
+- [GitHub Page](https://github.com/astoilkov/jsblocks)
```

Comment:
This seems like linking up an official page 4 times. Are there any interesting blog posts we can link up?